src/main/webapp/styles/general.css

```diff
@@ -90,6 +90,23 @@ input[type=submit]:hover {
   color: #4285f4;
 }
 
+button {
```

Comment:
There is a lot of overlap between button and input[type-submit] could you combine these?

Reply:
reorganized buttons

Reply:
Sorry if my initial comment wasnt clear i was thinking you would share the style with the buttons and the input[type=submit] in general.css like:

button, input[type=submit] {
style attributes
}

Also for the elements with the id account-functions, is that for a specific page? If so that should probably be in the CSS file for that page.

Reply:
Yeah I actually originally changed it like that but then i realized the input[type=submit] was not even being used so I just removed it all together